homeassistant/components/transmission/__init__.py

```diff
@@ -85,19 +88,39 @@ def setup(hass, config):
         hass, config, api)
     tm_data.init_torrent_list()
```

Comment:
Add a call to update the api here, as planned. Either insert it here, above `tm_data.init_torrent_list`, or inside `tm_data.init_torrent_list`
```py
tm_data.update()
```